### PR TITLE
Specify field names when initializing structs

### PR DESCRIPTION
**Pull request by pradykaushik**
Before submitting, please make sure you run a vagrant box running Aurora with the latest version shown in .auroraversion
and run go test from the project root.

To run an Aurora Vagrant image, follow the instructions here:
http://aurora.apache.org/documentation/latest/getting-started/vagrant/

* Have you run goformat on the project before submitting?

* Have you run go test on the project before submitting? Do all tests pass?

* Does the Pull Request require a test to be added to the end to end tests? If so, has it been added?

This pull request is related to Issue #45 
Ran `gofmt` on the entire project.

Ran `go test`. All the tests passed.
```
Create call took 13316244 ns
Kill call took 19764415 ns
Schedule cron call took 8676829 ns
Deschedule cron call took 21711283 ns
PASS
ok  	github.com/paypal/gorealis	0.161s
```

This pull request does not require any new tests to be added to the end to end tests.

### CODE REVIEW

File: examples/client.go [hunk 1]
```diff
@@ -348,7 +348,10 @@ func main() {
 			instId = k
 			break
 		}
-		resp, err := r.AddInstances(aurora.InstanceKey{job.JobKey(), instId}, numOfInstances)
+		resp, err := r.AddInstances(aurora.InstanceKey{
```
Comment (rdelval) [inline examples/client.go]: Push the second arg onto it's own line.
i.e:
```
resp, err := r.AddInstances(aurora.InstanceKey{
			JobKey:     job.JobKey(),
 			InstanceId: instId,
 		}, 
    numOfInstances)
```

File: examples/client.go [hunk 2]
```diff
@@ -437,7 +451,10 @@ func main() {
 		break
 	case "rollbackUpdate":
 		fmt.Println("Abort update")
-		resp, err := r.RollbackJobUpdate(aurora.JobUpdateKey{job.JobKey(), updateId}, "")
+		resp, err := r.RollbackJobUpdate(aurora.JobUpdateKey{
```
Comment (rdelval) [inline examples/client.go]: Same here

File: examples/client.go [hunk 3]
```diff
@@ -428,7 +439,10 @@ func main() {
 		break
 	case "abortUpdate":
 		fmt.Println("Abort update")
-		resp, err := r.AbortJobUpdate(aurora.JobUpdateKey{job.JobKey(), updateId}, "")
+		resp, err := r.AbortJobUpdate(aurora.JobUpdateKey{
```
Comment (rdelval) [inline examples/client.go]: Same here

File: examples/client.go (hunk 1 above)
Comment (pradykaushik) [reply examples/client.go]: done

File: examples/client.go (hunk 3 above)
Comment (pradykaushik) [reply examples/client.go]: done

File: examples/client.go (hunk 2 above)
Comment (pradykaushik) [reply examples/client.go]: done

**Pull request merged**